tutorials/katacoda/basic-pipeline/step1.md

```diff
@@ -0,0 +1,75 @@
+In this section, we will build the first task.
+
+With Tekton, each operation in your CI/CD workflow becomes a Step, which is executed with a container image you specify. Steps are then organized in Tasks, which run as a Kubernetes pod in your cluster. You can further organize Tasks into Pipelines, which can control the order of execution of several Tasks.
+
+To create a Task, create a Kubernetes object using the Tekton API with the kind Task. The following YAML file specifies a Task with one simple Step, which prints a Hello World! message using the official Ubuntu image:
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: echo
+spec:
+  steps:
+    - name: hello
+      image: ubuntu
+      command:
+        - echo
+      args:
+        - "Hello World!"
+```
+
+Write the YAML above to a file named task-hello.yaml, and apply it to your Kubernetes cluster:
```

Comment:
we should take advantage of the [`assets` config](https://www.katacoda.community/scenario-syntax.html#assets) to copy the sample files into the katacoda environment, or use the katacoda [markdown extensions](https://www.katacoda.community/scenario-syntax.html#execute) to allow users to click on the command to execute it

Reply:
I think i've added `{{copy}}` and `{{execute}}` where needed.